CMakeLists.txt

```diff
@@ -155,6 +156,13 @@ target_include_directories(fc
     ${CMAKE_CURRENT_SOURCE_DIR}/vendor/websocketpp
   )
 
+# try and make this very clear that this json parser is intended only for webauthn parsing..
```

Comment:
This seems overly punitive.  Is there a substantive reason that we would have to whitelist use of this json parser?

Reply:
Since JSON parsing is consensus I am concerned about the version of rapidjson being accidentally upgraded in the future and breaking that consensus. So, I tried to make it very clear this version of rapidjson is strictly for webauthn parsing: it's in a directory called `webauthn_json` and it can only be included from elliptic_webauthn.cpp.